### ISSUE TITLE
Build from source not working

### ISSUE THREAD

**Issue by RobinUS2** [williaster/data-ui#165]
Hi there,

I attempted a couple of builds from source (as instructed, npm install, lerna bootstrap). These however seem to fail at a variety of versions (tested master at a0165e0ee587a7e908826c2db15c46da6d8afc51, 0.69 at 0840a3b92c08dd792d28f96727ae71197b6e5e2c ). 

Errors vary, at master it is:
```
$lerna bootstrap


info cli using local version of lerna
lerna notice cli v3.11.1
lerna notice filter excluding "build-config"
lerna info filter [ '!build-config' ]
lerna info Bootstrapping 11 packages
lerna info Installing external dependencies
lerna info Symlinking packages and binaries
lerna info lifecycle @data-ui/data-table@0.0.75~prepublish: @data-ui/data-table@0.0.75
lerna info lifecycle @data-ui/theme@0.0.75~prepublish: @data-ui/theme@0.0.75
lerna info lifecycle @data-ui/forms@0.0.75~prepublish: @data-ui/forms@0.0.75

> @data-ui/data-table@0.0.75 prepublish /Users/someone/snip/data-ui/packages/data-table
> yarn run build

yarn run v1.13.0
$ yarn run build:cjs && yarn run build:esm
$ beemo babel ./src --out-dir lib/ --react --minify

Unknown blueprint for "configure". Must be a builder.
    at /Users/someone/snip/data-ui/packages/data-table/node_modules/@boost/core/node_modules/optimal/lib/optimal.js:32:13
    at Array.forEach (<anonymous>)
    at buildAndCheck (/Users/someone/snip/data-ui/packages/data-table/node_modules/@boost/core/node_modules/optimal/lib/optimal.js:23:26)
    at Object.optimal [as default] (/Users/someone/snip/data-ui/packages/data-table/node_modules/@boost/core/node_modules/optimal/lib/optimal.js:64:10)
    at ConfigLoader.loadConfig (/Users/someone/snip/data-ui/packages/data-table/node_modules/@boost/core/lib/ConfigLoader.js:201:41)
    at Tool.loadConfig (/Users/someone/snip/data-ui/packages/data-table/node_modules/@boost/core/lib/Tool.js:417:41)
    at Tool.initialize (/Users/someone/snip/data-ui/packages/data-table/node_modules/@boost/core/lib/Tool.js:302:14)
    at new Beemo (/Users/someone/snip/data-ui/packages/data-table/node_modules/@beemo/core/lib/Beemo.js:42:62)
    at Object.<anonymous> (/Users/someone/snip/data-ui/packages/data-table/node_modules/@beemo/cli/lib/CLI.js:21:15)
    at Module._compile (internal/modules/cjs/loader.js:734:30)

error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
``` 

at 0.69 
```
> beemo babel ./src --out-dir lib/ --minify


Invalid BabelDriver field "clean.alias". String cannot be empty.
    at StringBuilder.invariant (/Users/someone/snip/data-ui/packages/data-ui-theme/node_modules/optimal/lib/Builder.js:175:15)
    at StringBuilder.checkNotEmpty (/Users/someone/snip/data-ui/packages/data-ui-theme/node_modules/optimal/lib/StringBuilder.js:103:16)
    at /Users/someone/snip/data-ui/packages/data-ui-theme/node_modules/optimal/lib/Builder.js:293:55
    at Array.forEach (<anonymous>)
    at StringBuilder.runChecks (/Users/someone/snip/data-ui/packages/data-ui-theme/node_modules/optimal/lib/Builder.js:290:21)
    at /Users/someone/snip/data-ui/packages/data-ui-theme/node_modules/optimal/lib/UnionBuilder.js:85:21
    at Array.forEach (<anonymous>)
    at UnionBuilder.checkUnions (/Users/someone/snip/data-ui/packages/data-ui-theme/node_modules/optimal/lib/UnionBuilder.js:82:18)
    at /Users/someone/snip/data-ui/packages/data-ui-theme/node_modules/optimal/lib/Builder.js:293:55
    at Array.forEach (<anonymous>)
```

```
$lerna --version
3.11.1
``` 

Could you please help me out? Thanks!

**Comment by williaster** [comment]
Thanks for checking out `@data-ui` and reporting this @RobinUS2 !

This is a problem with an unpinned lib in our build-config. Should be fixed now after merging #166, let me know if you still have issues.

**Comment by RobinUS2** [comment]
Confirmed 👍 thanks for the swift reply.

```
lerna success Bootstrapped 11 packages
```
